README.md

```diff
@@ -257,7 +262,7 @@ git command:
 git pull
 ~~~
 However, for running the equivalent of `git pull` on all the repositories managed by
-the codyco-superbuild, you have to execute in your build system the appropriate target.
+the robotology-superbuild, you have to execute in your build system the appropriate target.
```

Comment:
There are still some references here and there about the word `codyco`. If you commit this change why not editing all the other references to it?

Reply:
Done in https://github.com/robotology/robotology-superbuild/pull/24/commits/358f52faa2ed34d37358c513638074b5d571314a .